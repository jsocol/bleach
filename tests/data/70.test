<isindex src="javascript:" onmouseover="alert(1)" label="bypass by @giutro" />
