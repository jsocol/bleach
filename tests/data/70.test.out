<form><hr><label>This is a searchable index. Enter search keywords: <input name="isindex" label="bypass by @giutro"></label><hr></form>
